packages/server/graphql/public/mutations/updateRecurrenceSettings.ts

```diff
@@ -1,51 +1,48 @@
-import ms from 'ms'
 import {SubscriptionChannel} from 'parabol-client/types/constEnums'
 import {RRule} from 'rrule'
 import getRethink from '../../../database/rethinkDriver'
 import {insertMeetingSeries as insertMeetingSeriesQuery} from '../../../postgres/queries/insertMeetingSeries'
 import restartMeetingSeries from '../../../postgres/queries/restartMeetingSeries'
+import updateMeetingSeriesQuery from '../../../postgres/queries/updateMeetingSeries'
 import {MeetingSeries} from '../../../postgres/types/MeetingSeries'
 import {analytics} from '../../../utils/analytics/analytics'
 import {getUserId, isTeamMember} from '../../../utils/authorization'
 import publish from '../../../utils/publish'
 import standardError from '../../../utils/standardError'
 import {MutationResolvers} from '../resolverTypes'
 
-const MEETING_DURATION_IN_MINUTES = 24 * 60 // 24 hours
-
-// Next meeting start is tomorrow at 9a UTC
-// :TODO: (jmtaber129): Determine this from meeting series configuration.
-const createNextMeetingStartDate = () => {
-  const now = new Date()
-  return new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() + 1, 9))
-}
-
-const startNewMeetingSeries = async (viewerId: string, teamId: string, meetingId: string) => {
+export const startNewMeetingSeries = async (
+  viewerId: string,
+  teamId: string,
+  meetingId: string,
+  meetingName: string,
+  recurrenceRule: RRule
+) => {
   const now = new Date()
   const r = await getRethink()
-  const nextMeetingStartDate = createNextMeetingStartDate()
-  const recurrenceRule = new RRule({
-    freq: RRule.WEEKLY,
-    byweekday: [RRule.MO, RRule.TU, RRule.WE, RRule.TH, RRule.FR],
-    dtstart: nextMeetingStartDate
-  })
+
+  // TODO: now, as new meeting series could be created from a new meeting view, let's grab the first part ie. "Standup - Jan 1" will be "Standup"
+  // TODO: this is a temporary solution, we should have a better way to handle this
+  const [cleanMeetingName] = meetingName.includes('-') ? meetingName.split('-') : [meetingName]
```

Comment:
+1 no need for the ternary, meetingName.split('-') works, or `meetingName.slice(0, meetingName.indexOf('-'))` plays better with typescript so you don't need to add a `!`